lib/rmt/cli/products.rb

```diff
@@ -36,41 +36,80 @@ def list
 
   desc 'enable TARGETS', _('Enable mirroring of product repositories by a list of product IDs or product strings.')
   option :all_modules, type: :boolean, desc: _('Enables all free modules for a product')
-  long_desc <<-REPOS
-#{_('Enable mirroring of product repositories by a list of product IDs or product strings.')}
+  long_desc <<~REPOS
+    #{_('Enable mirroring of product repositories by a list of product IDs or product strings.')}
 
-#{_('Examples')}:
+    #{_('Examples')}:
 
-$ rmt-cli products enable SLES/15
+    $ rmt-cli products enable SLES/15
 
-$ rmt-cli products enable 1575
+    $ rmt-cli products enable 1575
 
-$ rmt-cli products enable SLES/15/x86_64 1743
+    $ rmt-cli products enable SLES/15/x86_64 1743
 
-$ rmt-cli products enable --all-modules SLES/15
-REPOS
+    $ rmt-cli products enable --all-modules SLES/15
+  REPOS
   def enable(*targets)
     change_products(targets, true, options[:all_modules])
   end
 
   desc 'disable TARGETS', _('Disable mirroring of product repositories by a list of product IDs or product strings.')
-  long_desc <<-REPOS
-#{_('Disable mirroring of product repositories by a list of product IDs or product strings.')}
+  long_desc <<~REPOS
```

Comment:
nice :+1: 
is this supported in Ruby 2.5.0+?

Reply:
@digitaltom yes, this is supported since 2.3 [prooflink](https://ruby-doc.org/core-2.5.0/doc/syntax/literals_rdoc.html#label-Here+Documents)

Reply:
:+1: for squiggly heredoc. I was recently wondering if Ruby had something like Python's `dedent()` :slightly_smiling_face: